src/Scripting/CSharp/CSharpScript.cs

```diff
@@ -23,7 +26,23 @@ public static class CSharpScript
         /// <typeparam name="T">The return type of the script</typeparam>
         public static Script<T> Create<T>(string code, ScriptOptions options = null, Type globalsType = null, InteractiveAssemblyLoader assemblyLoader = null)
         {
-            return Script.CreateInitialScript<T>(CSharpScriptCompiler.Instance, code, options, globalsType, assemblyLoader);
```

Comment:
and add cod comment that this method throws ArgNullEx


---
In reply to: [99941131](https://github.com/dotnet/roslyn/pull/16489#discussion_r99941131) [](ancestors = 99941131)